lightspark-crypto-go/signing.go

```diff
@@ -0,0 +1,70 @@
+package lightspark_crypto
+
+// TODO(mhr): Add support for other OS.
+
+// #cgo darwin,amd64 LDFLAGS: -L${SRCDIR}/libs/darwin/amd64 -llightspark_crypto
+// #cgo darwin,arm64 LDFLAGS: -L${SRCDIR}/libs/darwin/arm64 -llightspark_crypto
+import "C"
+import (
+	"strings"
+
+	"github.com/lightsparkdev/lightspark-crypto-uniffi/lightspark-crypto-go/internal"
+)
+
+func GetMnemonicSeedPhrase(entropy []byte) ([]string, error) {
+	mnemonic, err := internal.MnemonicFromEntropy(entropy)
+	if err != nil {
+		return nil, err
+	}
+	defer mnemonic.Destroy()
```

Comment:
Ah, yeah this is a lot like the kotlin memory weirdness that caused me to write wrappers there. Makes sense.